README.md

```diff
@@ -22,17 +21,17 @@
 
 ## Installation
 
-If on a Mac without NPM/NodeJS version 16 installed, run:
+If on a Mac without NPM/Node.js version 16 installed, run:
 
 ```bash
 brew install npm
 brew install node@16
```

Comment:
I thought we upgraded from 16?

Reply:
We never tested for Node18. This is something that we need to test to ensure this works.

Reply:
AL2 only support Node16 right now